ros_cross_compile/platform.py

```diff
@@ -79,6 +85,8 @@ def __init__(
             self._ros_version = 'ros'
         else:
             raise ValueError('Unknown ROS distribution "{}" specified'.format(ros_distro))
+        if testing_version:
+            self._ros_version = '{}-testing'.format(self._ros_version)
```

Comment:
I don't think that passing a boolean is necessarily the best, it seems needlessly constraining users. Why can't users just pass one or more extra APT repos that would be added to the Docker sysroot image? That would prevent cross-compile from breaking if the repo structure changes.